mlrun/api/crud/model_endpoints.py

```diff
@@ -866,16 +876,33 @@ def deploy_model_monitoring_batch_processing(
         task = mlrun.new_task(name="model-monitoring-batch", project=project)
         task.spec.function = function_uri
 
+        # Apply batching interval params
+        minutes, hours, day, _, _ = tuple(
+            map(
+                lambda element: float(f"0{element.partition('/')[-1].strip(' ')}"),
+                tracking_policy[
+                    model_monitoring_constants.EventFieldType.DEFAULT_BATCH_INTERVALS
+                ].split(" "),
+            )
+        )
```

Comment:
Move to function and add unittests.

Reply:
Done. 

```
    @staticmethod
    def _get_batching_interval_param(intervals_list):
        """Converting each value in the intervals list into a float number. None
        Values will be converted into 0.0.
        example::
        Applying the function on a scheduling policy that is based on every hour exactly
        _get_batching_interval_param(intervals_list=[0, '*/1', None])
        The result will be: (0.0, 1.0, 0.0)
        """
        return tuple(
            map(
                lambda element: 0.0
                if isinstance(element, (float, int)) or element is None
                else float(f"0{element.partition('/')[-1]}"),
                intervals_list,
            )
        )
```

I have already added tests